pkg/antctl/raw/multicluster/common/common.go

```diff
@@ -213,6 +212,21 @@ func deleteServiceAccounts(cmd *cobra.Command, k8sClient client.Client, namespac
 	}
 }
 
+func ConvertMemberTokenSecret(secret corev1.Secret) corev1.Secret {
```

Comment:
Add some comments saying the result Secret struct is for generating the token Secret manifest for creating the Secret in a member cluster?